zebra-network/src/address_book.rs

```diff
@@ -453,7 +490,10 @@ impl AddressBook {
     fn update_metrics(&mut self, instant_now: Instant, chrono_now: chrono::DateTime<Utc>) {
         let _guard = self.span.enter();
 
-        let m = self.address_metrics(chrono_now);
+        let m = self.address_metrics_internal(chrono_now);
```

Comment:
Following here what i started in https://github.com/ZcashFoundation/zebra/pull/3244/files#r772018714 . Here the function is used in production code so the correctness change from https://github.com/ZcashFoundation/zebra/pull/3244/files#r772018714 will need to be changed.

Reply:
I added 03102da to explain the method better.